src/modules/ekf2/EKF/ekf_helper.cpp

```diff
@@ -1037,10 +937,19 @@ void Ekf::updateGroundEffect()
 	}
 }
 
+// do not call before quaternion states are initialised
```

Comment:
```suggestion
```

If this is only used in one place instead of warning the caller why don't we just inline it?